data/strategies/StrategysteCurveWETHSingleSided.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "strategy",
   "name": "StrategysteCurveWETHSingleSided",
-  "description": "Supplies WETH to the liquidity pool on Curve here to obtain steCRV tokens which it then puts into the v2 Curve stETH Pool yVault (yvsteCRV)to gain yield.",
+  "description": "Supplies WETH to the liquidity pool on Curve here to obtain steCRV tokens which it then puts into the v2 Curve stETH Pool yVault (yvsteCRV) to gain yield.",
```

Comment:
Why this is different from the Curve Voter Proxy strats? Doesn't it get the yield from CRV rewards?